documentation/pyserial_api.rst

```diff
@@ -164,10 +164,14 @@ Native ports
         :return: Bytes read from the port.
         :rtype: bytes
 
-        Read until an expected sequence is found ('\\n' by default), the size
-        is exceeded or until timeout occurs. If a timeout is set it may
-        return fewer characters than requested. With no timeout it will block
-        until the requested number of bytes is read.
+        Read until the expected *sequence* is found (``b'\n'`` by default),
+        or *size* bytes have been received, or ``timeout`` seconds have elapsed.
```

Comment:
Should "timeout" be rendered as code style?  This would be referring to the `*timeout*` argument passed to the constructor, but it's a bit unclear.

Maybe retain consistency with the rest of the docs (eg `read()` method above) and just leave it as plain text like " or the read timeout has elapsed".